app/controllers/series_controller.rb

```diff
@@ -107,20 +107,20 @@ def create
   # PUT /series/1
   # PUT /series/1.xml
   def update
-    unless params[:series][:author_attributes][:ids]
+    unless series_params[:author_attributes][:ids]
       flash[:error] = ts("Sorry, you cannot remove yourself entirely as an author of a series right now.")
       redirect_to edit_series_path(@series) and return
     end
-    
-    if params[:pseud] && params[:pseud][:byline] && params[:pseud][:byline] != "" && params[:series][:author_attributes]
+
+    if params[:pseud] && params[:pseud][:byline] && params[:pseud][:byline] != "" && series_params[:author_attributes]
       valid_pseuds = Pseud.parse_bylines(params[:pseud][:byline])[:pseuds] # an array
       valid_pseuds.each do |valid_pseud|
-        params[:series][:author_attributes][:ids] << valid_pseud.id rescue nil
+        series_params[:author_attributes][:ids] << valid_pseud.id rescue nil
```

Comment:
As long as the specs pass - I just pushed up a fix that should hopefully resolve both issues. I also re-enabled the test for series co-authors since it should be fixed.